src/test/scala/is/hail/methods/KeyTableSuite.scala

```diff
@@ -512,10 +512,9 @@ class KeyTableSuite extends SparkSuite {
 
   @Test def testGroup() {
     val data = Array(
-      Array("Sample1", Row(23.0f, "rabbit"), 9, 5),
-      null)
```

Comment:
I don't now what this null was doing here.  Top-level row structure in KeyTables can't be missing.